svg-injector.js

```diff
@@ -141,23 +145,11 @@
             //
             // :NOTE: IE8 and older doesn't have DOMParser, but they can't do SVG either, so...
             else if (DOMParser && (DOMParser instanceof Function)) {
-              var xmlDoc;
-              try {
-                var parser = new DOMParser();
-                xmlDoc = parser.parseFromString(httpRequest.responseText, 'text/xml');
-              }
-              catch (e) {
-                xmlDoc = undefined;
-              }
-
-              if (!xmlDoc || xmlDoc.getElementsByTagName('parsererror').length) {
+              var parsedSvg = parseSvg(httpRequest.responseText);
+              if(parsedSvg)
```

Comment:
when submitting a PR, it's good to keep author's coding style:
- please correct the spacing
- please add the curlies